c++/src/kj/map.h

```diff
@@ -77,6 +77,8 @@ class HashMap {
   kj::Maybe<Value&> find(KeyLike&& key);
   template <typename KeyLike>
   kj::Maybe<const Value&> find(KeyLike&& key) const;
+  template <typename KeyLike>
+  bool contains(KeyLike&& key) const;
```

Comment:
Should this really be inserted here before the code comment below? The code comment applies to the `find(...)` methods. Does it also apply to this new `contains(...)` method? If not, then `contains(...)` should be moved to after the comment.